modules/react-native-status/android/build.gradle

```diff
@@ -16,7 +16,9 @@ dependencies {
     implementation 'com.instabug.library:instabug:3+'
     implementation 'status-im:function:0.0.1'
```

Comment:
@jakubgs Where would we get this `status-im:function` module from? It is currently in artifactory.

Reply:
If you look closer you can see that I've added `android/app/libs/function-0.0.1.jar` to the repo itself. It's a tiny file.